diambraArena/diambraEnvLib/diambra_pb2.py

```diff
@@ -12,12 +12,9 @@
 _sym_db = _symbol_database.Default()
 
 
-
```

Comment:
This file is autogenerated, so we shouldn't reformat it

Reply:
you mean the whole file?